pwiz_tools/Skyline/Model/Lib/BiblioSpec.cs

```diff
@@ -154,12 +159,18 @@ public override void WriteXml(XmlWriter writer)
                 writer.WriteAttribute(ATTR.score, Score.Value);
                 writer.WriteAttribute(ATTR.score_type, ScoreType);
             }
+            if (!string.IsNullOrEmpty(Protein))

```

Comment:
You can use "WriteAttributeIfString" here.

Reply:
Spiffy.